demos/sample_azure_iot_adu/sample_azure_iot_adu.c

```diff
@@ -155,9 +156,17 @@ static uint8_t ucCommandResponsePayloadBuffer[ 256 ];
 static uint8_t ucReportedPropertiesUpdate[ 1500 ];
 static uint32_t ulReportedPropertiesUpdateLength;
 
-static uint8_t ucAduContextBuffer[ 1024 ];
+static uint8_t ucAduContextBuffer[ 10240 ];
```

Comment:
ooof that's a big buffer. it is what it is :)

Reply:
I was oooooverly lax with the sizes for now. Next we will need to trim all these buffers.